packages/inetutils.rb

```diff
@@ -36,9 +39,36 @@ def self.build
       --enable-authentication \
       --disable-servers"
     system 'make'
+    system "cat <<'EOF'> ping_
```

Comment:
These wrappers can be written without using `system`, which looks a bit nicer. For example:
```
PING_SH = <<EOF
#!/bin/bash
sudo -E #{CREW_PREFIX}/sbin/capsh --caps='cap_net_raw+eip cap_setpcap,cap_setuid,cap_setgid+ep' \\
    --keep=1 --user=nobody --addamb=cap_net_raw -- \\
    -c "#{CREW_PREFIX}/bin/ping.elf \$@"
EOF
IO.write('ping_', PING_SH, perm: 0755)
```

In the example above, I define `PING_SH` and then immediately use it, but IMO it should be moved out of `self.build` for readability.

Reply:
Why not go ahead and write to `#{CREW_DEST_PREFIX}/bin/ping`?

Reply:
@uberhacker because ```system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'``` will overwrite ```#{CREW_DEST_PREFIX}/bin/ping```.

Reply:
Thanks @kjperkin. Do these changes look correct to you?

Reply:
@uberhacker Moved the heredocs to install, where as per your suggestion they get written straight to ```#{CREW_DEST_PREFIX}/bin/```.

Reply:
@satmandu looking better. I read up on [heredocs](https://ruby-doc.org/core-3.0.0/doc/syntax/literals_rdoc.html#label-Here+Documents+-28heredocs-29) a bit more and if you put a `~` before the EOF, then you can indent everything nicely. I also think it would be good to move the constants outside of `self.install` so it's easier to read the method.

Reply:
Example of how you would change the constants?